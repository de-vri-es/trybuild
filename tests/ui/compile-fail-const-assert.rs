fn main() {
    const { assert!(false) };
}
