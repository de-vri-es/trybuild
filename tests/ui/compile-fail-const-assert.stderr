error[E0080]: evaluation of `main::{constant#0}` failed
 --> tests/ui/compile-fail-const-assert.rs:2:13
  |
2 |     const { assert!(false) };
  |             ^^^^^^^^^^^^^^ the evaluated program panicked at 'assertion failed: false', $DIR/tests/ui/compile-fail-const-assert.rs:2:13
  |
  = note: this error originates in the macro `assert` (in Nightly builds, run with -Z macro-backtrace for more info)

note: erroneous constant encountered
 --> tests/ui/compile-fail-const-assert.rs:2:5
  |
2 |     const { assert!(false) };
  |     ^^^^^^^^^^^^^^^^^^^^^^^^
